doc/src/ds-test-tutorial.md

```diff
@@ -98,7 +98,7 @@ contract MyContract is Test {
 }
 ```
 
-After compiling with `forge build`, when ran under hevm, we get:
+After compiling with `forge build --ast`, when ran under hevm, we get:
```

Comment:
I think there is also probably some config file option that forge supports to force this? It might be a nicer workflow to recomend that people set that in the tutorial?

Reply:
Good point! Thanks, `GPT-40` allowed me to find this config setting! Weirdly enough, the `Phind-70B` model told me it doesn't exist. I am now closing & merging. Thanks again!